newrelic/readme.md

```diff
@@ -8,36 +8,44 @@
 
 ### DESCRIPTION
 
-This document describes SignalFx's integration with [New Relic](https://www.newrelic.com). Use this integration to view New Relic metrics in SignalFx. With this integration, you can apply SignalFx's analytics to New Relic data, use New Relic data in alerts, and enrich your infrastructure data with data about your New Relic-instrumented applications.  
+This document describes SignalFx's integration with [New Relic](https://www.newrelic.com). Use this integration to view New Relic metrics in SignalFx. With this integration, you can apply SignalFx's analytics to New Relic data, use New Relic data in alerts, and enrich your infrastructure data with data about your New Relic-instrumented applications.
 
 ### REQUIREMENTS AND DEPENDENCIES
 
-This integration requires a New Relic account. You must be an administrator of your SignalFx organization to create or modify this integration. 
+This integration requires a New Relic account. You must be an administrator of your SignalFx organization to create or modify this integration.
 
 ### INSTALLATION
 
-1. Follow [New Relic's instructions](https://docs.newrelic.com/docs/apis/rest-api-v2/requirements/api-keys) to obtain a REST API key for your account. 
+1. Follow [New Relic's instructions](https://docs.newrelic.com/docs/apis/rest-api-v2/requirements/api-keys) to obtain a REST API key for your account.
 
-1. Click the **New Integration** button, and in the new field labelled **API key** enter the API key that you obtained in step 1. 
+1. Click the **New Integration** button. In the new field labelled **API key** enter the API key that you obtained in step 1, then click **Validate**.
 
-1. Optionally, select the individual New Relic modules for which you want metrics to appear in SignalFx. By default, SignalFx collects data from APM, Mobile, and Servers modules. 
+1. A message appears that says **Validated!**. If a different message appears, contact support@signalfx.com for help.
 
```

Comment:
Does this need to be a separate step or can it be stated as the result of the previous step? 

(I'm editing for brevity here because every linebreak pushes the config form further down.)